### ISSUE TITLE
The actionscript task detection didn't contribute a task for the following configuration

### ISSUE THREAD

**Issue by atmofunk** [BowlerHatLLC/vscode-as3mxml#533]
With 1.7, i started getting this (the title of this post) for all my build tasks.

my task:
```
{
	"label": "Steam Build",
	"air": "windows",
	"debug": false,
	"type": "actionscript",
	"asconfig": "asconfig.steam-debug.json",
	"group": "build",
	"problemMatcher": []
}
```

I have been using this config for a year now -- no files have changed locations, everything is as it was last week.   but now when i try to run this *nothing* happens, except for the error

Did something change with 1.7 that i need to update ?

**Comment by atmofunk** [comment]
user error -- sorry.  closing.

**Comment by joshtynjala** [comment]
I can actually reproduce this error, and there is definitely a bug.

**Comment by atmofunk** [comment]
Now i'm really confused because I restarted Windows (pending update) and the problem disappeared.

**Comment by joshtynjala** [comment]
Strange! Well, I just released v1.7.1, which fixes the bug that I found.

**Comment by atmofunk** [comment]
Well i'm glad my weird issue-but-not-really-? led to something positive at least!
